apps/examples/src/examples/custom-renderer/README.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Custom renderer
 component: ./CustomRendererExample.tsx
-category: basic
+category: ui
```

Comment:
not sure if i agree with "ui" categorization for this one but i can't think of a better one.

I kinda hate this example though and would be down to set `hide: true` on this one

Reply:
There are definitely some people I speak to who want that sort of thing. The category ID is "ui" but the title of that category is "UI & theming" so I think it works okay here. I also considered having this one and some others in an "Advanced" category - all themed around using tldraw as an engine. But that felt too vague